src/containers/BuildUnsignedSweepCoin/ArbethTokenForm.tsx

```diff
@@ -0,0 +1,163 @@
+import { Form, FormikHelpers, FormikProvider, useFormik } from 'formik';
+import { Link } from 'react-router-dom';
+import * as Yup from 'yup';
+import { Button, FormikTextfield } from '~/components';
+
```

Comment:
Sorry I missed previously, but actually we can merge these 2 token files as one in the Erc20TokenForm by renaming it to EthLikeTokenForm.tsx. Please take reference of EthLikeForm.tsx. Any variable values can be added to allCoinMetas

Reply:
done